cpp/ql/src/semmle/code/cpp/ir/implementation/raw/internal/TranslatedCall.qll

```diff
@@ -324,6 +324,15 @@ class TranslatedFunctionCall extends TranslatedCallExpr, TranslatedDirectCall {
   override predicate hasWriteSideEffect() {
     not expr.getTarget().(SideEffectFunction).hasOnlySpecificWriteSideEffects()
   }
+
+  override Instruction getQualifierResult() {
+    hasQualifier() and
+    result = getQualifier().getResult()
+  }
+
+  override predicate hasQualifier() {
+    not exists(MemberFunction func | expr.getTarget() = func and func.isStatic())
```

Comment:
Doesn't this hold for all non-member functions as well?

Reply:
Good point! I've fixed this in 6c7e1cd. There are no changes in the tests as `TranslatedFunctionCall::getQualifierResult()` would fail for non-member functions anyway.